MekHQ/resources/mekhq/resources/Mission.properties

```diff
@@ -3,6 +3,58 @@
+AtBContractType.SECURITY_DUTY.text=Security Duty
+AtBContractType.SECURITY_DUTY.toolTipText=<html>The unit is tasked with protecting a specific person, object, or place from attack and/or capture. <br>This is also called "bodyguard" or "watchdog" duty.</html>
+AtBContractType.RIOT_DUTY.text=Riot Duty
+AtBContractType.RIOT_DUTY.toolTipText=The unit is tasked with garrisoning a region to put down and prevents riots and rebellion.
+AtBContractType.PLANETARY_ASSAULT.text=Planetary Assault
+AtBContractType.PLANETARY_ASSAULT.toolTipText=The unit is tasked with assaulting a planet or a region of a planet, normally as part of a larger force.
+AtBContractType.RELIEF_DUTY.text=Relief Duty
+AtBContractType.RELIEF_DUTY.toolTipText=The unit is tasked with providing relief and reinforcement to embattled employer units, normally ones in danger of being overrun without the additional support.
+AtBContractType.GUERRILLA_WARFARE.text=Guerrilla Warfare
+AtBContractType.GUERRILLA_WARFARE.toolTipText=The unit is tasked with going behind enemy lines without support to undertake guerrilla actions against the enemy, thus disrupting their planetary control.
+AtBContractType.PIRATE_HUNTING.text=Pirate Hunting
+AtBContractType.PIRATE_HUNTING.toolTipText=The unit is tasked with hunting down pirates and brining them to justice... or death.
+AtBContractType.DIVERSIONARY_RAID.text=Diversionary Raid
+AtBContractType.DIVERSIONARY_RAID.toolTipText=The unit is tasked with providing a diversion for a separate, far more important, assault or raid by raiding an enemy planet, thus drawing significant enemy forces to their position from the true primary target.
+AtBContractType.OBJECTIVE_RAID.text=Objective Raid
+AtBContractType.OBJECTIVE_RAID.toolTipText=The unit is tasked with raiding an enemy planet to achieve a set military objective.
+AtBContractType.RECON_RAID.text=Recon Raid
+AtBContractType.RECON_RAID.toolTipText=The unit is tasked with scouting and gathering information on an enemy force by raiding an enemy planet.
+AtBContractType.EXTRACTION_RAID.text=Extraction Raid
+AtBContractType.EXTRACTION_RAID.toolTipText=The unit is tasked with raiding an enemy planet to capture a target and return the target to their employer.
+
+# AtBLanceRole Enum
+AtBLanceRole.FIGHTING.text=Fight
+AtBLanceRole.FIGHTING.toolTipText=The lance has been tasked with performing offensive actions against the enemy.
+AtBLanceRole.DEFENCE.text=Defend
+AtBLanceRole.DEFENCE.toolTipText=The lance has been tasked with performing defensive actions, protecting designated targets and locations from the enemy.
+AtBLanceRole.SCOUTING.text=Scout
+AtBLanceRole.SCOUTING.toolTipText=The lance has been tasked with performing reconnaissance actions, scouting the enemy and their positions.
+AtBLanceRole.TRAINING.text=Training
+AtBLanceRole.TRAINING.toolTipText=The lance has been tasked with trained very green and green members of the force.
```

Comment:
"very green and green" seems pretty redundant.

Reply:
It should be ultra green and green, as those are the two skill levels that are trained. I've also added the required officer skill level, as the goal is to explain a bit about the role within MekHQ.